pkg/modules/checks/standalone.go

```diff
@@ -61,7 +66,9 @@ func (s *Standalone) ValidateOpValues() modules.UserOpHandlerFunc {
 		g.Go(func() error { return ValidateVerificationGas(ctx.UserOp, s.maxVerificationGas) })
 		g.Go(func() error { return ValidatePaymasterAndData(ctx.UserOp, gc, gs) })
 		g.Go(func() error { return ValidateCallGasLimit(ctx.UserOp) })
-		g.Go(func() error { return ValidateFeePerGas(ctx.UserOp, gbf) })
+		if !isBSC(ctx.ChainID) {
+			g.Go(func() error { return ValidateFeePerGas(ctx.UserOp, gbf) })
+		}
```

Comment:
I don't think switching off this sanity check based on hardcoded chain ID values is the best approach for this. Instead we should have a fallback mechanism if chains don't support EIP-1559 style gas fees.

Reply:
Ok, how could we know programmatically if chains don't support  EIP-1559 style gas fees apart from knowing their chain IDs?
And this fallback mechanism should be part of `ValidateFeePerGas()` function or somewhere else?